scene/gui/scroll_bar.cpp

```diff
@@ -497,6 +494,22 @@ Size2 ScrollBar::get_minimum_size() const {
 	return minsize;
 }
 
+void ScrollBar::scroll(double p_amount) {
+	double prev_scroll = get_value();
+	set_value(prev_scroll + p_amount);
+	if (!Math::is_equal_approx(prev_scroll, get_value())) {
+		emit_signal(SNAME("scrolling"));
+	}
+}
+
+void ScrollBar::scroll_to(double p_position) {
+	double prev_scroll = get_value();
+	set_value(p_position);
+	if (!Math::is_equal_approx(prev_scroll, get_value())) {
```

Comment:
You can change this to compare `get_value()` with `p_position` and do early return.

Reply:
Done. Will be in next push.